lib/saml_idp/metadata_builder.rb

```diff
@@ -24,41 +24,41 @@ def fresh
-            sign entity
-
-            entity.IDPSSODescriptor protocolSupportEnumeration: protocol_enumeration do |descriptor|
-              build_key_descriptor descriptor
-              build_name_id_formats descriptor
-              descriptor.SingleSignOnService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
-                Location: single_service_post_location
-              descriptor.SingleSignOnService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect",
-                Location: single_service_post_location
-              build_attribute descriptor
-            end
-
-            entity.AttributeAuthorityDescriptor protocolSupportEnumeration: protocol_enumeration do |authority_descriptor|
-              build_key_descriptor authority_descriptor
-              build_organization authority_descriptor
-              build_contact authority_descriptor
-              authority_descriptor.AttributeService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect",
-                Location: attribute_service_location
-              build_name_id_formats authority_descriptor
-              build_attribute authority_descriptor
-            end
-
-            build_organization entity
-            build_contact entity
+                                 xmlns: Saml::XML::Namespaces::METADATA,
+                                 'xmlns:saml' => Saml::XML::Namespaces::ASSERTION,
+                                 'xmlns:ds' => Saml::XML::Namespaces::SIGNATURE,
+                                 entityID: entity_id do |entity|
+          sign entity
+
+          entity.IDPSSODescriptor protocolSupportEnumeration: protocol_enumeration do |descriptor|
+            build_key_descriptor descriptor
+            build_name_id_formats descriptor
+            descriptor.SingleSignOnService Binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST',
+                                           Location: single_service_post_location
+            descriptor.SingleSignOnService Binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+                                           Location: single_service_post_location
+            build_attribute descriptor
           end
+
+          entity.AttributeAuthorityDescriptor protocolSupportEnumeration: protocol_enumeration do |authority_descriptor|
+            build_key_descriptor authority_descriptor
+            build_organization authority_descriptor
+            build_contact authority_descriptor
+            authority_descriptor.AttributeService Binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+                                                  Location: attribute_service_location
+            build_name_id_formats authority_descriptor
+            build_attribute authority_descriptor
+          end
+
+          build_organization entity
+          build_contact entity
+        end
       end
     end
-    alias_method :raw, :fresh
+    alias raw fresh
```

Comment:
more `alias_method`

Reply:
fixed in https://github.com/18F/saml_idp/pull/98/commits/2b8c1afa357e64a05ddf8932dfae209da2a40a6a